app/models/grda_warehouse/client_notes/alert.rb

```diff
@@ -13,15 +13,13 @@ def self.type_name
     end
 
     # anyone who can see this client
-    scope :visible_by, -> (user, client) do
+    scope :visible_by, ->(user, client) do # rubocop:disable Lint/UnusedBlockArgument
```

Comment:
Is the disable any different than renaming the parameter `_client`? If not, I think I'd prefer the name change.